notificationsManager/Operators/Database/Entities/Operator.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+namespace NotificationsManager\Operators\Database\Entities;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="operators")
+ */
+class Operator
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     */
+    protected int $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    protected int $customerId;
+
+    /**
+     * @ORM\Column(type="string", length=100)
+     */
+    protected string $name;
+
+    /**
+     * @ORM\Column(type="string", length=100, nullable=true)
+     */
+    protected string $surname_1;
+
+    /**
+     * @ORM\Column(type="string", length=100, nullable=true)
+     */
+    protected string $surname_2;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    protected int $phone;
+
+    /**
+     * @ORM\Column(type="string", length=100, nullable=true)
+     */
+    protected string $email;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    protected bool $orderNotificationsEnabled;
+
+    /**
+     * @ORM\Column(type="string", length=100, nullable=true)
+     */
+    protected string $orderNotificationsEmail;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    protected bool $orderNotificationsByEmail;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    protected bool $orderNotificationsBySms;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    protected bool $orderNotificationsByPush;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    protected bool $deleted;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    protected datetime $createdAt;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    protected datetime $updatedAt;
+
+    /**
+     * @param mixed $id
+     */
```

Comment:
Los comentarios de params solo los usamos para castear los tipos de los arrays por ejemplo User[].

Lo que puedes hacer es:
```suggestion
     public function setId(int $id): void
    {
        $this->id = $id;
    }
```